---
permalink: /
title: "About Me"
author_profile: true
redirect_from:
  - /about/
  - /about.html
---

I am a 2nd-year PhD candidate at the Department of Surgery ([https://www.med.cuhk.edu.hk/](https://www.surgery.cuhk.edu.hk/), CUHK Medical School, where I develop computation pipelines and machine learning approaches for cancer genomics under the supervision of [Prof. Xin WANG](https://xinwlab.netlify.app/). My research focuses on:

- **Tumor Heterogeneity Analysis**: Specifically, tumor alternative splicing-induced protein isoform large-scale foldome analysis.
- **Biomarker Discovery**: Through integrative multi-omics approaches.
- **Clinical Predictive Modeling**: Utilizing deep learning architectures.
- **Computation-aided Drug Design**: Developing reconstruction algorithms and methods for computation-aided drug (molecules and antibody) screening and design.

---

### Contact

**Email**: tjiang@surgery.cuhk.edu.hk

---

### Google Scholar

[https://scholar.google.com/citations?user=_YT1sPQAAAAJ&hl=en&oi=sra](https://scholar.google.com/citations?user=_YT1sPQAAAAJ&hl=en&oi=sra)

---

### Summary of Recent Works

- **Chromosomal-Level Genome Assembly of a True Bug, *Aspongopus chinensis* Dallas, 1851 (Hemiptera: Dinidoridae)**
  T Jiang, Z Yin, R Cai, H Yu, Q Lu, S Zhao, Y Tian, Y Yan, J Guo, X Chen
  *Genome Biology and Evolution 13 (10), evab232*

- **MetaGXplore: Integrating Multi-Omics Data with Graph Convolutional Networks for Pan-cancer Patient Metastasis Identification**
  T Jiang, H Jiang, X Ma, M Xu, Y Liang, W Zhang

---

### Projects

Here are some of the key open-source projects I've been leading:

<p align="center">
<table>
  <tr>
    <td align="center">
      <a href="http://hkwanglab-compbio.com:3831/">
        <img src="https://github.com/user-attachments/assets/4a4482e7-d5b8-4507-afca-ff1521371d32" width="300"><br>
        <b>3DisoGalaxy</b>
      </a>
    </td>
    <td align="center">
      <a href="https://github.com/CityUHK-CompBio/3Diso-TransFold">
        <img src="https://github.com/user-attachments/assets/c60c5cb0-f865-4689-9130-476feb545052" width="300"><br>
        <b>3DisoTransFold</b>
      </a>
    </td>
    <td align="center">
      <a href="https://github.com/FeliciaJiangBio/3DisoTarget"> 
        <img src="https://github.com/user-attachments/assets/5d7aab79-7eec-4ae3-9a0a-8a3e8722fb2e" width="300"><br> 
        <b>3DisoTarget</b>
      </a>
    </td>
  </tr>
</table>
</p>
![3DisoTarget]

---

### Selected Awards

For site content, there is one Markdown file for each type of content, which are stored in directories like `_publications`, `_talks`, `_posts`, `_teaching`, or `_pages`. For example, each talk is a Markdown file in the [_talks directory](https://github.com/academicpages/academicpages.github.io/tree/master/_talks). At the top of each Markdown file is structured data in YAML about the talk, which the theme will parse to do lots of cool stuff. The same structured data about a talk is used to generate the list of talks on the [Talks page](https://academicpages.github.io/talks), each [individual page](https://academicpages.github.io/talks/2012-03-01-talk-1) for specific talks, the talks section for the [CV page](https://academicpages.github.io/cv), and the [map of places you've given a talk](https://academicpages.github.io/talkmap.html) (if you run this [python file](https://github.com/academicpages/academicpages.github.io/blob/master/talkmap.py) or [Jupyter notebook](https://academicpages.github.io/blob/master/talkmap.ipynb), which creates the HTML for the map based on the contents of the `_talks` directory).

---

### Conference Attending

- ISMB (2025)
